hackathon_site/dashboard/frontend/src/pages/Dashboard/Dashboard.js

```diff
@@ -21,6 +21,28 @@ import {
 } from "testing/mockData";
 import { hackathonName } from "constants.js";
 
+import SideSheetRight from "components/general/SideSheetRight/SideSheetRight";
+import ProductOverview from "components/general/ProductOverview/ProductOverview";
+
+const testD = {
```

Comment:
1. Give this a better name than `testD`
2. If you'd like to use this, put it under `testing/mockData.js` and import it with the rest, above